velox/functions/prestosql/Arithmetic.h

```diff
@@ -229,8 +229,7 @@ struct PowerFunction {
   template <typename TInput>
   FOLLY_ALWAYS_INLINE void
   call(double& result, const TInput& a, const TInput& b) {
-    result =
-        std::isnan(b) ? std::numeric_limits<double>::quiet_NaN() : pow(a, b);
+    result = 0.0;
```

Comment:
afaik you have to change the signature of the function, not it's implementation for the fuzzer to get triggered:
![image](https://github.com/user-attachments/assets/9f4e6b65-70f9-4311-9a4c-5998b97e1d5a)

Reply:
Ah, good catch. I added a new signature. Thanks!

Reply:
Nice, it triggered the build correctly!  There is an error where I am not sure if this is just because of the mock change or if something is not quite working: https://github.com/facebookincubator/velox/actions/runs/11919376847/job/33221625324?pr=11581#step:7:70

Reply:
Hi @assignUser, I've fixed that issue. Now the mock change triggers a fuzzer failure due to result mismatch from Presto: https://github.com/facebookincubator/velox/actions/runs/11926331226/job/33241036209?pr=11581.

Reply:
Great!